spec.html

```diff
@@ -44553,7 +44553,7 @@ <h1>
           1. Let _methodContext_ be the running execution context.
           1. Suspend _methodContext_.
           1. Set _generator_.[[GeneratorState]] to ~executing~.
-          1. Push _genContext_ onto the execution context stack; _genContext_ is now the running execution context.
+          1. <emu-meta effects="user-code">Push _genContext_ onto the execution context stack</emu-meta>; _genContext_ is now the running execution context.
```

Comment:
Nit: I'd put this one the following step. The "Resume" is where the actual transfer of control happens.

Ditto below.

Reply:
Done.